builtin/providers/openstack/resource_openstack_networking_network_v2_test.go

```diff
@@ -104,6 +107,81 @@ func TestAccNetworkingV2Network_netstack(t *testing.T) {
 	})
 }
 
+func TestAccNetworkingV2Network_fullstack(t *testing.T) {
+	region := os.Getenv(OS_REGION_NAME)
+
+	var instance servers.Server
+	var network networks.Network
+	var port ports.Port
+	var secgroup secgroups.SecurityGroup
+	var subnet subnets.Subnet
+
+	var testAccNetworkingV2Network_fullstack = fmt.Sprintf(`
+    resource "openstack_networking_network_v2" "foo" {
+      region = "%s"
+      name = "network_1"
+      admin_state_up = "true"
+    }
+
+    resource "openstack_networking_subnet_v2" "foo" {
+      region = "%s"
+      name = "subnet_1"
+      network_id = "${openstack_networking_network_v2.foo.id}"
+      cidr = "192.168.199.0/24"
+      ip_version = 4
+    }
+
+    resource "openstack_compute_secgroup_v2" "foo" {
+      region = "%s"
+      name = "secgroup_1"
+      description = "a security group"
+      rule {
+        from_port = 22
+        to_port = 22
+        ip_protocol = "tcp"
+        cidr = "0.0.0.0/0"
+      }
+    }
+
+    resource "openstack_networking_port_v2" "foo" {
+      region = "%s"
+      name = "port_1"
+      network_id = "${openstack_networking_network_v2.foo.id}"
+      admin_state_up = "true"
```

Comment:
Would it make sense to also allow none-string `true`? I was surprised that it doesn't work with boolean value